test/tests/functional/pbs_mom_dynamic_resource.py

```diff
@@ -70,6 +70,17 @@ def create_mom_resources(self, resc_name, resc_type, resc_flag,
             self.filenames.append(fp)
         return fp_list
 
+    def check_log(self, fp, exist=True):
+        match_from = time.time()
+        # adding a second delay because log_match can then start from the
```

Comment:
Is the 1 second delay really needed? If the log message will be generated by hup'ing the mom, then you should set match_from after hup'ing it, and then that should find the correct log message, no?